src/components/ConfirmationMessage/index.jsx

```diff
@@ -6,6 +6,7 @@ export const ConfirmationMessage = ({
   onConfirm,
   children,
   confirmationButtonContent = 'Continue',
+  confirmationButtonType = 'checkmark',
```

Comment:
It would be much better if we replace `confirmationButtonPositive` with two values: `confirmationButtonPositive` and `confirmationButtonNegative`:

```js
import React from 'react'
import { Button, Modal, Header } from 'semantic-ui-react'

export const ConfirmationMessage = ({
  onClose,
  onConfirm,
  children,
  confirmationButtonContent = 'Continue',
  confirmationButtonPositive = true,
  confirmationButtonNegative = false,
}) => {
  return (
    <Modal closeIcon open size="tiny" onClose={onClose}>
      <Header icon="warning sign" content="Warning" />
      <Modal.Content>
        <Modal.Description>{children}</Modal.Description>
      </Modal.Content>
      <Modal.Actions>
        <Button secondary onClick={onClose} content="Close" />
        <Button
          positive={confirmationButtonPositive}
          negative={confirmationButtonNegative}
          onClick={onConfirm}
          content={confirmationButtonContent}
        />
      </Modal.Actions>
    </Modal>
  )
}
```

And after that update `ContractCardForCustomer`:
<img width="996" alt="Screenshot 2023-02-18 at 2 24 10 PM" src="https://user-images.githubusercontent.com/335095/219855321-b27b2a1d-828d-4ecb-b3a9-d5c1a5eb1c36.png">

Also we should change `ContractCardForContractor` and add `confirmationButtonPositive` to `ConfirmationMessage`.